acceptance/tests/allow_arbitrary_node_name_for_apply.rb

```diff
@@ -1,5 +1,9 @@
 test_name "node_name_value should be used as the node name for puppet apply"
 
+tag 'audit:medium',
+    'audit:integration',  # Ruby level integration tests already exist. This acceptance test can be deleted.
+    'audit:delete'
```

Comment:
any reason this PR shouldn't just delete this test?  i had thought about using an audit:delete tag as well.  i think our progress tooling can count the number of tests and display that near the job runtime